MathPortal/MathPortal/Source/Data/User.swift

```diff
@@ -11,6 +11,13 @@ import Parse
 
 class User: ParseObject {
     
+    static var current: User? {
```

Comment:
This doesn't look too good. Every time you access `User.current` you would create a new user instance and update it? Are you sure?